pkg/cmd/enclave_configs_logs.go

```diff
@@ -96,7 +95,7 @@ var configsLogsRollbackCmd = &cobra.Command{
 			utils.HandleError(err.Unwrap(), err.Message)
 		}
 
-		if !silent {
+		if !utils.Silent {
```

Comment:
No really related to this PR i think, but instead having to wrap this around everywhere we print, it would be easier if we just have this in the `printer.ConfigLog(` method or a wrapper around this so that we don't need this if statements every/potentially forget about it.

Reply:
I considered making that change when updating all of these. Then I realized that we sometimes want our print functions to print regardless of `utils.Silent`, so I opted not to change for now.